src/HasStates.php

```diff
@@ -140,6 +140,27 @@ public function transitionTo($state, string $field = null)
         $this->{$field}->transitionTo($state);
     }
 
+    /**
+     * @param string $fromClass
+     * @param string|null $field
+     */
+    public function transitionableStates(string $fromClass, string $field = null)
```

Comment:
Add array return type and remove docblocks, also use `?string $field = null`, since it's semantically more correct.